stardis/radiation_field/opacities/opacities_solvers/base.py

```diff
@@ -368,6 +368,7 @@ def calc_alpha_line_at_nu(
         # add ionization energy to lines
         ionization_data = stellar_plasma.ionization_data.reset_index()
         ionization_data["ion_number"] -= 1
+        ionization_data["ion_number"] = ionization_data["ion_number"]
```

Comment:
Isn't this self-assignment?

Reply:
Yup, good catch, thanks. This was left over from some type issues I was debugging.